docs/sources/panels/transformations/prereq.md

```diff
@@ -0,0 +1,23 @@
++++
+title = "Prerequisites"
```

Comment:
Since this content is in another topic, this topic can go away.

Reply:
Removed topic.